content/systems/filecoin_mining/sector/sealing.md

```diff
@@ -1,18 +1,43 @@
 ---
 title: Sector Sealing
+weight: 4
 dashboardWeight: 2
-dashboardState: wip
+dashboardState: stable
 dashboardAudit: wip
 dashboardTests: 0
 ---
 
 # Sector Sealing
 
-{{<embed src="sealing.id" lang="go" >}}
+Before a Sector can be used, the Miner must _seal_ the Sector: encode the data in the Sector to prepare it for the proving process.
+
+* **Unsealed Sector**: A Sector of raw data.
+    * **UnsealedCID (CommD)**: The root hash of the Unsealed Sector's merkle tree. Also called CommD, or "data commitment."
+* **Sealed Sector**: A Sector that has been encoded to prepare it for the proving process.
+    * **SealedCID (CommR)**: The root hash of the Sealed Sector's merkle tree. Also called CommR, or "replica commitment."
+
+Sealing a sector through Proof-of-Replication (PoRep) is a computation-intensive process that results in a unique encoding of the sector. Once data is sealed, storage miners: generate a proof; run a SNARK on the proof to compress it; and finally, submit the result of the compression to the blockchain as a certification of the storage commitment. Depending on the PoRep algorithm and protocol security parameters, cost profiles and performance characteristics vary and tradeoffs have to be made among sealing cost, security, onchain footprint, retrieval latency and so on. However, sectors can be sealed with commercial hardware and sealing cost is expected to decrease over time. The Filecoin Protocol will launch with Stacked Depth Robust (SDR) PoRep with a planned upgrade to Narrow Stacked Expander (NSE) PoRep with improvement in both cost and retrieval latency.
+
+The Lotus-specific set of functions applied to the sealing of a sector can be found [here](https://github.com/filecoin-project/lotus/blob/master/storage/sealing.go).
+
+## Randomness
+
+Randomness is an important attribute that helps the network verify the integrity of Miners' stored data. Filecoin's block creation process includes two types of randomness:
+* [DRAND](#drand): Values pulled from a distributed random beacon
```

Comment:
Does this work? You should use ref to files plus anchors if needed not anchor alone.